geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml

```diff
@@ -634,204 +635,201 @@ vars:
         relevant_for:
           - layer_wms
       - name: geometryValidation
         type: boolean
         description: >
-          Force validation of the geometries according to the layer type (point, line, polygon),
-          for example, to prevent creation of a 2-point polygon.
+          Enable geometry validation according to layer type (point, line, polygon).
+          For example, prevents creation of 2-point polygons.
         relevant_for:
           - layer_wms
       - name: customOpenLayersOptions
         type: json
         description: >
-          The custom OpenLayers WMS layer options.
+          Custom OpenLayers options for WMS layers.
         relevant_for:
           - layer_wms
           - layer_wmts
           - layer_cog
       - name: editingAttributesOrder
         type: list
         description: >
-          Specify the order of the editable attributes in the edit interface.
+          Specify the order of editable attributes in the edit interface.
         relevant_for:
           - layer_wms
       - name: editingEnumerations
         type: json
         description: >
-          Specify the value column and the order column on the relationship's target table of the
-          enumeration.
+          Specify value column and order column for enumeration relationships.
         relevant_for:
           - layer_wms
       - name: readonlyAttributes
         type: list
         description: >
-          Set attributes as read only in the Editing form.
+          Coma separated list of attributes to make read-only in the editing form.
         relevant_for:
           - layer_wms
       - name: queryIconPosition
         type: list
         description: >
-          Values to define the shape (bbox) to use to query the layer. The values are used like a
-          padding in css with 1, 2, 3 or 4 comma separated values: all / top-bottom, left-right /
-          top, right-left, bottom / top, right, bottom, left.
+          Define query shape (bbox) padding values similar to CSS:
+          <code>all</code> or <code>top-bottom, left-right</code> or <code>top, right-left, bottom</code>
+          or <code>top, right, bottom, left</code>.
         relevant_for:
           - layer_wms
           - layer_wmts
           - layer_cog
       - name: editingSelectionAttribute
         description: >
-          The field used in the selector tooltip to name the layer when multiple ones are selected on the map.
+          Field shown in selector tooltip to identify the layer when multiple layers are selected.
         relevant_for:
           - layer_wms
       # WMS
       - name: directedFilterAttributes
         type: list
         description: >
-          List of attribute names which should have rules already ready when using the filter
-          tools.
+          Coma separated list of attributes names which should have rules already ready when using the
```

Comment:
"Coma" is not correct, must be "Comma" 
Same comment for other occurrences in the PR

Reply:
+1 Thanks :-)